server/node/server.js

```diff
@@ -29,9 +28,34 @@ app.get("/", (req, res) => {
   res.sendFile(path);
 });
 
-app.get("/public-key", (req, res) => {
-  res.send({ publicKey: process.env.STRIPE_PUBLISHABLE_KEY });
-});
+app.get("/setup-page", asyncMiddleware(async (req, res, next) => {
+    animals = process.env.ANIMALS.split(',');
+
+    var lookup_keys = [];
+    animals.forEach(animal => lookup_keys.push(animal + "-monthly-usd"));
+
+    const prices = await stripe.prices.list({
+      lookup_keys: lookup_keys,
+      expand:["data.product"]});
+
+    var products = [];
+    prices.data.forEach(price => {
+      var product = {
+        price: {id: price.id, unit_amount: price.unit_amount},
+        title: price.product.metadata.title,
+        emoji: price.product.metadata.emoji
+      };
+      products.push(product);
+    });
```

Comment:
We should probably migrate all samples that use products to something similar. This is great :)